### ISSUE TITLE
Make it model agnostic

### ISSUE THREAD

**Issue by ocefpaf** [ocefpaf/yam2ncml#3]
For now it is specific to ROMS.

**Comment by rsignell-usgs** [comment]
Yeah, I guess we can look forward to `--roms`, `--fvcom`, `--selfe`, etc.

**Comment by ocefpaf** [comment]
This is something we need to think before re-factoring.  One idea is to ship templates for `roms`, `fvcom`, `selfe`, etc and load them using a mandatory argument like `--model=<model_arg>`.

Ee can treat the current implementation as a `roms` template, factor it out, and providing a model for future templates.

**Comment by ocefpaf** [comment]
Instead of `--model_name` we can do that via templates. See #31.
